docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -157,104 +149,94 @@ cp sample.env .env
 copy sample.env .env
 ```
 
-Then you have to install the dependencies required for the application to startup.
-
-```shell
-# Install NPM dependencies
-npm install
-```
+The keys in the `.env` file are *not* required to be changed to run the app locally. You can leave the default values copied over from `sample.env` as-is.
 
-The keys are not required to be changed, to run the app locally. You can leave the default values from the `sample.env` as is.
+Keep in mind if you want to use additional services, you'll have to get your own API keys for those services and edit those entries accordingly in the `.env` file.
 
+#### Step 2: Install dependencies
+This step will install the dependencies required for the application to run:
 
-Next, let's bootstrap the various services, i.e. the api-server, the client UI application, etc. You can [learn more about these services in this guide](#).
-
-`MONGOHQ_URL` is the most important one. Unless you have MongoDB running in a setup different than the defaults, the URL in the `sample.env` should work fine.
-
-You can leave the other keys as they are. Keep in mind if you want to use more services you'll have to get your own API keys for those services and edit those entries accordingly in the `.env` file.
-
-### Start MongoDB
+```sh
+# Install NPM dependencies
+npm ci
```

Comment:
One concern that I have with this, subsequent installs.

That is if someone does add a new package or updates a package as a part of the a legitimate change, `npm ci` will blow away the `node_modules`.

This would mean instead of an update it would become more slower to update the local setup. But that just circles around to the original problem.

Reply:
It blows it away but it is very fast.  I would recommend we do some testing though if there are concerns

Reply:
Yup, I did some research. And it's going to be fine. Since all it does is look at the lockfile which in turn has a version of a package pinned, it means that almost every time the package is going to be there in the cache that npm has already created on your local machine.

Let's proceed. I am quite happy with this change.